src/pyscanner.l

```diff
@@ -1049,6 +1050,8 @@ STARTDOCSYMS      "##"
 
                           yyextra->current->section = Entry::CLASS_SEC;
                           yyextra->current->name = yytext;
+                          // we need tos set the protectiion based on the "l.cal" class name
```

Comment:
typo? `l.cal` &rarr; `local` (I suppose)

Reply:
Indeed, was a second typo as well `tos` -> `to`